bundles/org.openhab.io.homekit/README.md

```diff
@@ -686,263 +678,267 @@ or using UI
-| IrrigationSystem     |                             |                              |                               | An accessory that represents multiple water valves and accommodates a programmed scheduled. |
-|                      | Active                      |                              | Switch                        | If the irrigation system as a whole is enabled. This must be ON if any of the valves are also enabled. |
-|                      | InUseStatus                 |                              | Switch                        | If the irrigation system as a whole is running. This must be ON if any of the valves are ON. |
-|                      | ProgramMode                 |                              | String                        | The current program mode of the irrigation system. Possible values (NO_SCHEDULED - no programs scheduled, SCHEDULED - program scheduled, SCHEDULED_MANUAL - program scheduled, currently overriden to manual mode). |
-|                      |                             | RemainingDuration            | Number                        | The remaining duration for all scheduled valves in the current program in seconds. |
-|                      |                             | FaultStatus                  | Switch, Contact               | Accessory fault status.  "ON"/"OPEN" value indicates that the accessory has experienced a fault that may be interfering with its intended functionality. A value of "OFF"/"CLOSED" indicates that there is no fault.                                                                                                                                |
-
+## Supported accessory types
+
+For configuration options, the default values are in parentheses.
+For enum values, the parentheses indicate the default values if the item is a Number or a Switch.
+All enum values can be customized via item metadata. I.e. `HEAT="heating", COOL="cooling"`, or `HEAT=5, COOL=7` for a Number.
+<a id="customizeable-enum">Some enums can have the list of valid values customized, meaning that if you customize the mapping, any value that is missing will not be presented to the user.</a>
+They are appropriately marked.
+Enums that are linked to Switches or Contacts have an `inverted` param that will reverse the sense of `ON`/`OFF` or `OPEN`/`CLOSED`.
+
+| Accessory Tag        | Mandatory Characteristics   | Optional Characteristics    | Supported openHAB item types            | Description                                                                                                                                                                                                                                                                                                                                                   | Configuration Options                                                 | Valid Enum Values                                                                                           |
```

Comment:
The main changes in this table are:
 * adding additional columns for configuration options and enum values, and moving any references to those things in the description column as appropriate, adopting a consistent formatting and ordering for enum values, as well as item types.
 * Sorting the entire table by accessory type name, then required and optional characteristics, and then characteristic name
 * Ensure all relevant item types are listed for enums (and they're sorted), as appropriate (I didn't list Switch or Contact for enums that didn't seem to have a reasonable use case for only using the first two values, even though technically the code would allow it)